modules/dynamic-provisioning-azure-file-definition.adoc

```diff
@@ -56,7 +56,7 @@ parameters:
   skuName: Standard_LRS <3>
   storageAccount: <storage-account> <4>
 reclaimPolicy: Delete
-volumeBindingMode: Immediate
+volumeBindingMode: WaitForFirstConsumer <5>
```

Comment:
Based on [this comment](https://bugzilla.redhat.com/show_bug.cgi?id=1918270#c2) in the BZ.

Reply:
Same as the last comment.

Reply:
Same question as previous, should this revert back to `Immediate`?

Reply:
Also reverted to `Immediate` here to maintain consistency in this field for Azure file docs.